level.py

```diff
@@ -1,44 +1,79 @@
 import ast
+from typing import Optional, List, Dict
+
+class Level(object):
+    NULL_EVENT = (-1, None)
 
-class Level():
     def __init__(self, filename: str=''):
-        self._events = {}
+        self._current_event = 0
+        self._events = []
 
         self._level_name = ''
         self._num_floors = 0
-        self._total_time_sec = 0.
+        self._capacity = 0
+        self._level_finish_move_number = 0
 
         if filename != '':
             self.parse_from_file(filename)
 
-    # todo implement interface for getting
+    def get_num_floors(self) -> int:
+        return self._num_floors
+
+    def get_capacity(self) -> int:
+        return self._capacity
+
+    def get_next_event(self, move: int) -> Optional[Dict[int, List[int]]]:  #TODO type hinting for null event?
```

Comment:
Isn't this type-hinting wrong anyway given that we are currently actually returning a tuple of move and event (I think we should just return event)?

Reply:
What is "event"?? Should that not be a dict like `{floors: [destinations_of_new_passengers]}`?

Reply:
Right, currently that is just `Dict[int, List[int]]` and is what I mean by event, could be extracted to a class of course but not really too bothered right now until basically it starts making tests hard to read

Reply:
Ah cool. not sure about the tuple then but know we had to fix this up to get the None's to work so @michaelDemertzi  presumably the checking is working??